collections/_services/certbot.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: service
+type: service
+
+title: Certbot
+version: ''
+order: 6
+description: >-
+   Use Certbot to automatically update TLS Certificates on OpenShift Routes.
+descriptionLinks:
+  - link:
+      url: https://github.com/BCDevOps/certbot
+      text: Github
+  - link:
+      url: https://github.com/BCDevOps/certbot/blob/master/README.md
+      text: Review Certbot documentation
+contentStyle: unstructured
+onboard:
+  - 'Self Hosted Service'
+  - 'DockerHub'
+onboardDescription: We provide a <em>Docker Image</em> as well as documented <em>open-source code</em> for hosting your own service.
+urls:
+  github: https://github.com/BCDevOps/certbot
+  docs: https://github.com/BCDevOps/certbot/blob/master/README.md
+dockerImages:
+  - name: Certbot
+    url: https://hub.docker.com/r/bcgovimages/certbot/
+    description: This image provides ???
+    badges:
+      - https://img.shields.io/docker/v/bcgovimages/certbot.svg?sort=semver
+      - https://img.shields.io/docker/pulls/bcgovimages/certbot.svg
+      - https://img.shields.io/docker/image-size/bcgovimages/certbot.svg
+---
+
+#### Feature List
+
+- Can utilize [https://letsencrypt.org/](https://letsencrypt.org/){:target="_blank"} or other ACME compliant Certificate Authority for issuing certificates
```

Comment:
maybe it's worth mentioning Entrust ?

Can utilize LetsEncrypt or other ACME compliant Certificate Authorities like [Entrust](https://www.entrust.com/digital-security/certificate-solutions/products/digital-certificates/tls-ssl-certificates){:target="_blank"}